<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProject</class>
 <widget class="QDialog" name="NewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>559</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="projectNameLabel">
          <property name="font">
           <font>
            <family>MS UI Gothic</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Project Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="projectNameEdit">
          <property name="font">
           <font>
            <family>MS Gothic</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>sample project</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="projectLocationLabel">
          <property name="font">
           <font>
            <family>MS UI Gothic</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Project Location:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="projectLocationEdit">
          <property name="font">
           <font>
            <family>MS Gothic</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileLocation">
          <property name="font">
           <font>
            <family>MS Gothic</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
